README.md

```diff
@@ -1,14 +1,20 @@
-# Windows Service Wrapper in less restrictive license
+# Windows Service Wrapper in a permissive license
```

Comment:
Nice! Should also port it to 2.x

Reply:
I added `*` to messages of other commits that I think could be ported to 2.x.